sdks/DeviceSDK.ts

```diff
@@ -0,0 +1,109 @@
+import { Inject, Singleton } from "typescript-ioc";
+import SettingSDK from './SettingSDK';
+import * as _ from "lodash";
+import { logger } from "@project-sunbird/logger";
+import SystemSDK from "./SystemSDK";
+import { HTTPService } from "@project-sunbird/ext-framework-server/services";
+import uuid = require("uuid");
+import axios from "axios";
+import jwt from "jsonwebtoken";
+import { DataBaseSDK } from "./DataBaseSDK";
+
+@Singleton
+export default class DeviceSDK {
+    @Inject private settingSDK: SettingSDK;
+    @Inject private systemSDK: SystemSDK;
+    @Inject private databaseSdk: DataBaseSDK;
+    private masterKey: string;
+
+    initialize(masterKey: string) {
+        this.masterKey = masterKey;
```

Comment:
rename it to config and create interface as object like
initialize(config: object) { 
this.config = config
}